packages/schema/breadboard.schema.json

```diff
@@ -1,178 +1,299 @@
-            "type": "string"
-          },
-          "in": {
-            "description": "The input port of the `to` node.\nMust be empty string or undefined if and only if `out` is either \"*\" or itself empty string or undefined.",
-            "type": "string"
-          },
-          "optional": {
-            "description": "If true, nodes connected to this edge won't wait for data to appear before proceding with execution.",
-            "type": "boolean"
-          },
-          "constant": {
-            "description": "If true, the most recent data that passed through this edge will remain available indefinitely, instead of being destructively consumed.",
-            "type": "boolean"
-          }
-        },
-        "oneOf": [
-          {
-            "required": [],
-            "properties": {
-              "out": {
-                "const": ""
-              },
-              "in": {
-                "const": ""
-              }
-            }
-          },
-          {
-            "required": [
-              "out"
-            ],
-            "properties": {
-              "out": {
-                "const": "*"
-              },
-              "in": {
-                "const": ""
-              }
-            }
-          },
-          {
-            "required": [
-              "out",
-              "in"
-            ],
-            "properties": {
-              "out": {
-                "$ref": "#/$defs/identifier"
-              },
-              "in": {
-                "$ref": "#/$defs/identifier"
-              }
-            }
-          }
-        ]
-      }
-    },
-    "kits": {
-      "description": "All of the kits this graph depends on",
-      "type": "array",
-      "items": {
-        "description": "A library that will be imported prior to execution for providing handlers for non built-in node types.",
-        "type": "object",
-        "required": [
-          "url"
-        ],
-        "additionalProperties": false,
-        "properties": {
-          "url": {
-            "description": "Address of the kit",
-            "type": "string",
-            "format": "uri"
-          }
-        }
-      }
-    },
-    "graphs": {
-      "description": "Sub-graphs that can be referred to by nodes in the parent graph.",
-      "type": "object",
-      "additionalProperties": false,
-      "patternProperties": {
-        "^[a-zA-Z_][a-zA-Z0-9_-]*$": {
-          "$ref": "#"
-        }
-      }
-    },
-    "args": {
-      "description": "Arguments that are passed to the graph, useful to bind values to lambdas.",
-      "type": "object",
-      "additionalProperties": true
-    }
-  }
+	"$id": "https://raw.githubusercontent.com/breadboard-ai/breadboard/@google-labs/breadboard-schema@1.2.0/packages/schema/breadboard.schema.json",
+	"$schema": "http://json-schema.org/draft-07/schema#",
```

Comment:
I used the $schema version at https://json-schema.org/learn/getting-started-step-by-step#create-a-schema-definition, I don't actually know about the different versions, but could you confirm that this new one we've switched to is the best/latest one to use?

Reply:
the majority with useful features are still draft. 2020 is newer. but still unsupported in many places. which was the result of a lot of my issues before. draft 07 is a lot more established